transgene.py

```diff
@@ -633,22 +607,27 @@ def run_transgene():
     This will try to run transgene from system arguments
     """
     parser = argparse.ArgumentParser(description=main.__doc__)
-    parser.add_argument('--peptides', dest='input_file',
-                        type=argparse.FileType('r'), help='Input peptide' +
-                                                          ' FASTA file', required=True)
+    parser.add_argument('--peptides', dest='peptide_file',
+                        type=argparse.FileType('r'),
+                        help='Path to GENCODE translation FASTA file')
```

Comment:
why is this no longer a required argument? Is it because you want to be able to run just the fusion module as a standalone? can you document this in the main docstring that is printed upon saying transgene --help?